airflow-core/src/airflow/ui/src/layouts/Details/Gantt/Gantt.tsx

```diff
@@ -114,7 +114,7 @@ export const Gantt = ({ limit }: Props) => {
     },
     undefined,
     {
-      enabled: Boolean(dagId),
+      enabled: Boolean(dagId) && runId !== "",
```

Comment:
Style nit:

```suggestion
      enabled: Boolean(dagId) && Boolean(runId),
```

I think? (Either way, they should both the same I think)

Reply:
Thanks!